test/scripts/generate-imported-targets-from-snyk.test.ts

```diff
@@ -179,35 +91,6 @@ describe('Generate imported targets based on Snyk data', () => {
     expect(importedTargetsLog).toMatch(targets[0].integrationId);
   }, 240000);
 
-  it('succeeds to generate targets for Group for Azure', async () => {
-    const logFiles = generateLogsPaths(__dirname, ORG_ID);
```

Comment:
same?

Reply:
same here we already have a test above for Azure, just removing testing it for org vs Group as this test takes a long time